core/domain/event_services_test.py

```diff
@@ -58,8 +58,8 @@ class EventHandlerTaskQueueUnitTests(test_utils.GenericTestBase):
     def test_events_go_into_the_events_queue(self):
         self.assertEqual(self.count_jobs_in_taskqueue(), 0)
 
-        event_services.StartExplorationEventHandler.record(
-            'eid1', 1, 'sid1', 'session1', {}, feconf.PLAY_TYPE_NORMAL)
+        event_services.CompleteExplorationEventHandler.record(
```

Comment:
Using `StartExplorationEventHandler` makes this test to fail since we're using the `exp_services.get_exploration_by_id` [here](https://github.com/oppia/oppia/pull/2238/files#diff-304548acca6cfab0c216a48f90dc1e4cR335), and the datastore fetch fails with an error!